src/net.cpp

```diff
@@ -1034,6 +1056,12 @@ bool CConnman::AttemptToEvictConnection()
 
     if (vEvictionCandidates.empty()) return false;
 
+    // If any remaining peers are preferred for eviction consider only them.
+    if (std::any_of(vEvictionCandidates.begin(),vEvictionCandidates.end(), [](NodeEvictionCandidate const & n) { return n.prefevict;})) {
```

Comment:
Should fix formatting.

Reply:
It isn't clear what you're asking for here.

Reply:
Mainly spaces inconsistencies.